pkg/controller/import-controller.go

```diff
@@ -106,7 +112,7 @@ func (ic *ImportController) processPvcItem(pvc *v1.PersistentVolumeClaim) error
 		}
 		// all checks passed, let's create the importer pod!
 		ic.expectPodCreate(pvcKey)
-		pod, err = CreateImporterPod(ic.clientset, ic.image, ic.verbose, ic.pullPolicy, ep, secretName, pvc)
+		pod, err = CreateImporterPod(ic.clientset, ic.image, ic.verbose, ic.pullPolicy, ep, secretName, resizeTo, pvc)
```

Comment:
We have found the need to change the number of parameters to a struct so we don't end up with 10 different parameters, please rebase to match.